src/config/production.json

```diff
@@ -2,6 +2,7 @@
 	"css_url": "https://widgets.wp.com/happychat/happychat.css",
 	"happychat_url": "https://happychat.io/customer",
 	"twemoji_cdn_url": "https://s0.wp.com/wp-content/mu-plugins/wpcom-smileys/twemoji/2/",
+	"oauth_client_id": "54006",
```

Comment:
I still need to understand this whole PR, but giving a first few thoughts:

* This number needs to be the app ID the host registers for Happychat. The [54006](https://developer.wordpress.com/apps/54006/) OAuth app only whitelists specific hosts for testing purposes: `localhost:900`, `woocommerce.test`, `calypso:localhost:300`, etc. This shouldn't ever ship to production because production host won't be whitelisted, ergo: it won't work.

* Until now, hosts were responsible for authentication, and the library only needed a specific token on behalf of the host. If we move authentication to the library, the `oauth_client_id` needs to be exposed in the API for production. Otherwise, hosts won't be able to use the library unless they clone this repo, modify the `oauth_client_id` number, and compile themselves the code.

Reply:
This was added only for the standalone example to work, based on private convos we will remove the regular wpcom oauth and make the example use tokens instead.